test/verify/main.cpp

```diff
@@ -81,8 +80,18 @@ int main(int argc, const char* argv[])
 #else
                 "batch_quant_dot_1<migraphx::fp8::fp8e4m3fnuz, float>",
                 "quant_dot_3args_4<migraphx::fp8::fp8e4m3fnuz, float>",
-                "quant_dot_3args_5<migraphx::fp8::fp8e4m3fnuz, float>"
+                "quant_dot_3args_5<migraphx::fp8::fp8e4m3fnuz, float>",
 #endif
+            "test_block_reduce_small<3, migraphx::shape::int8_type>",
+            "test_block_reduce_small<4, migraphx::shape::int8_type>",
+            "test_block_reduce_small<8, migraphx::shape::int8_type>",
+            "test_block_reduce_small<16, migraphx::shape::int8_type>",
```

Comment:
These works for GCC ? I had to insert #ifdefs earlier.

Reply:
It doesnt matter if its gcc or clang. We convert it to a float on the cpu backend which produces different results than just using int8.